livemaker/patch.py

```diff
@@ -89,17 +89,17 @@ def bar_show(item):
                 for info in bar:
                     # data = orig_lm.read(info, decompress=False)
                     # new_lm.writebytes(info, data)
-                    if info.path == lsb_path:
+                    if os.path.isfile(info.path):#if file exists in current folder
                         # replace existing with patch version, use original
                         # compress type
-                        new_lm.write(lsb_path, compress_type=info.compress_type)
-                        log.info('patched {}'.format(lsb_path))
+                        new_lm.write(info.path, compress_type=info.compress_type)
+                        print('patched {}'.format(info))
```

Comment:
This should still be logging calls and not prints.
It should already supposed to be set up to log info level messages to a file, and warnings/errors to a file, but if it's not working then that's a bug for me to fix I guess.
I think my original intention for logging was to add a `verbose` command line option that would also send info level to stdout, but I didn't get around to adding that in, but ideally we should not be spamming what could potentially be every file in the archive to stdout if the user doesn't explicitly specify that they want it.